### ISSUE TITLE
ReferenceError: variable not defined

### ISSUE THREAD

**Issue by abijr** [AnWeber/vscode-httpyac#82]
On vscode with httpyac versions 4.4.0 and 4.5.0, running the first block works, but running the second one throws an error: `ReferenceError: user not defined.`

Going back to httpyac version 4.3.0, allows me to run the same with the variables interpolated as expected.
```
{{
    exports.user = "user",
    exports.password = "password"
}}
GET /?test={{user}}

### Auth
POST /login
Content-Type: application/json

{
    "email": "{{ user }}",
    "password": "{{ password }}"
}

{{
    exports.user_id = response.parsedBody.user_id;
}}
````

**Comment by AnWeber** [comment]
Same error report like https://github.com/AnWeber/httpyac/issues/89 and root of change is https://github.com/httpyac/httpyac.github.io/issues/7
I scoped the variables per region/ request in the file and not per http file. The goal is that the request behaves identically on the first call compared to a call after other requests. I would have intended to use @ref explicitly, this would also have the advantage that the login script does not have to be called manually. But since the behavior differs from rest-client, I have now hidden it behind a configuration switch `useRegionScopedVariables`.

**Comment by AnWeber** [comment]
The bug has already been fixed with version 4.5.0. If there are still problems, please let us know.
